modules/ROOT/pages/licenses.adoc

```diff
@@ -23,7 +23,7 @@ Each time a case starts, the case counter is increased by one. +
 Case-counter licenses are available for development and for production. A development license must not be used for production, and typically has a lower case limit than a production license. +
 The maximum number of cases is defined in your commercial contract, in discussion with your sales person.
```

Comment:
I thought the number of everything was unlimited, no?

Reply:
There is a maximum number of cases, I think, we display that in the License page
![image](https://user-images.githubusercontent.com/22596322/133444971-0d32458f-743e-4647-a46e-71f65272d9b2.png)

Reply:
Since I never used Bonita in production, idk how it works

Reply:
ok